_posts/2019-02-20-lecture-notes-template.md

```diff
@@ -0,0 +1,361 @@
+To perform inference on our topic models, we start by considering the joint likelihood based on
+the hidden and observed random variables. 
+Leveraging the graph structure we have for topic models, we can factorize this 
+distribution in the fashion we are used to.
+
+$$P(\beta, \theta, z, w) = \Pi_{k=1}^{K}P(\beta_k | \eta)\Pi_{d = 1}^{D}p(\theta_d | \alpha)
+   \Pi_{n = 1}^{N}p(z_{d_n}|\theta_d)p(w_{d_n} | z_{d_n}, \beta)$$
+
+Given a query, answering the above joint would require marginalizing out the variables and 
+values that do no interest us - but doing so would require super-exponential work in this model,
+integrating across variables that may have a large set of possible values. Thus, our typical 
+approach is not tractable here.
+
+This motivates approximate inference: we trade off exact computation for reasonable computational
+work. We will cover two major families of methods in this lecture and next:
+Variational Inference and Markov Chain Monte Carlo.
+
+
+## Variational Inference
+
+Variational inference is a technique that allows one to convert an inference problem into an
+optimization problem. We want to maximize the data likelihood, but as we already discussed, 
+that itself is too hard to handle. Thus, we begin by lower-bounding the data likelihood
+with a term known as the "free-energy".
+
+
+We start by introducing another distribution , $Q_{\theta}$ which is a distribution on the 
+same set of random variables. The goal, eventually, is to have $Q_{\theta}$ be a distribution
+that is easier to work with than $p$ but is sufficiently "close" to serve as a reasonable 
+surrogate in calculations. 
+
+Using this, we want to maximize the lower bound for the log-likelihood:
+
+$$log(p(x)) = 
+KL(q_{\theta}(z | x) || p_{\theta}(z | x)) + \int_z q_{\theta}(z | x)log\frac{p_{\theta}(x, z)}{q_{\theta}(z | x)}dz 
+\ge \int_z q_{\theta}(z | x)log\frac{p_{\theta}(x, z)}{q_{\theta}(z | x)}dz
+:= \mathscr{L}(\theta, \phi ; x)$$
+
+Equivalently, we can minimize the aforementioned "free-energy" of the system:
+
+
+$$F(\theta, \phi ; x) = -log(p(x)) + KL(q_{\theta}(z | x) || p_{\theta}(z | x))$$
+
+Intuitively, the connection between $$\mathscr{L}(\theta, \phi ; x)$$ and $$F(\theta, \phi ; x)$$ is that the $$KL$$ divergence measures the gap between the lowerbound on the likelyhood ($$\mathscr{L}$$) and the real likliehood ($$log(p(x))$$) - both the minimization and the maximization noted above try to close that gap.
+
+
+We call $$\mathscr{L}(\theta, \phi ; x)$$ above the variational lower bound.
+Often it is written as $$\mathscr{L}(\theta, \phi ; x) = log(p(x)) - KL(q_{\theta}(z | x) || p_{\theta}(z | x))$$, which simply is $$\mathscr{L}(\theta, \phi ; x) = -F(\theta, \phi ; x)$$ .
+
+
+
+## Mean-Field Assumption (In Topic Models)
+
+Recall the form of the true posterior :
+
+$$p(\beta, \theta, z | w) = \frac{p(\beta, \theta, z, w)}{p(w)}$$
+
+Suppose that in q (our approximation to p) we could break dependancies in the jiont by assuming
```

Comment:
$q$, $p$, jiont is a typo

Reply:
I am not sure what you mean by this. This is exactly the form provided by Dr. Xing. Q was not conditional here.